render/topologies.go

```diff
@@ -116,6 +118,56 @@ var ContainerRenderer = MakeReduce(
 	),
 )
 
+type containerWithHostIPsRenderer struct {
+	Renderer
+}
+
+// Render produces a process graph where the ips for host network mode are set
+// to the host's IPs.
+func (r containerWithHostIPsRenderer) Render(rpt report.Report) RenderableNodes {
+	containers := r.Renderer.Render(rpt)
+	hosts := MakeMap(
+		MapHostIdentity,
+		SelectHost,
+	).Render(rpt)
+
+	for id, c := range containers {
+		networkMode, ok := c.Node.Latest.Lookup(docker.ContainerNetworkMode)
+		if !ok || networkMode != docker.NetworkModeHost {
```

Comment:
Could do. That is a bit more complicated, as you can do `container:weavescope`, `container:a1b2c3d4`, or `container:a1b2c3d4e5f6g7h8i9j0`, so we need to parse apart the network mode string, and lookup the container by name/id/short-id. It already sort-of works (it only shows one of the IPs). I'd say let's open a different issue for that.